api/src/main/java/com/capitalone/dashboard/service/IssueService.java

```diff
@@ -0,0 +1,59 @@
+package com.capitalone.dashboard.service;
+
+import com.capitalone.dashboard.model.Issue;
+import com.capitalone.dashboard.model.DataResponse;
+import com.capitalone.dashboard.request.IssueRequest;
+
+public interface IssueService {
+
+    /**
+     * Finds all of the Issues matching the specified request criteria.
+     *
+     * @param request search criteria
+     * @return Issues matching criteria
+     */
+    DataResponse<Iterable<Issue>> search(IssueRequest request);
+
+    /**
+     * Finds all of the Issues matching the specified request criteria.
+     *
+     * @param request search criteria
+     * @return Issues matching criteria
+     */
+    DataResponse<Iterable<Issue>> searchClosed(IssueRequest request);
+
+    /**
+     * Finds all of the Issues matching the specified request criteria.
+     *
+     * @param request search criteria
+     * @return Issues matching criteria
+     */
+    DataResponse<Iterable<Issue>> searchOpen(IssueRequest request);
+
+    /**
+     * Finds all of the Issues matching the specified request criteria.
+     *
+     * @param request search criteria
+     * @return Issues matching criteria
+
+
+    DataResponse<Iterable<Issue>> searchByOrgs(IssueRequest request);
+
+    /**
+     * Finds all of the Issues matching the specified request criteria.
+     *
+     * @param request search criteria
+     * @return Issues matching criteria
+
+    DataResponse<Iterable<Issue>> searchClosedByOrgs(IssueRequest request);
+
+    /**
+     * Finds all of the Issues matching the specified request criteria.
+     *
+     * @param request search criteria
+     * @return Issues matching criteria
+
+    DataResponse<Iterable<Issue>> searchOpenByOrgs(IssueRequest request);
+    */
+
+}
```

Comment:
Refactor Issues to gitIssues? So that there is no confusion about Jira issues etc?